message/message.py

```diff
@@ -261,9 +260,10 @@ def loadNestedEvent(self, event, context):
     * @param {*} event The input message sent to the Lambda
     * @returns {*} message that is ready to pass to an inner task
     """
-    config = self.__loadConfig(event, context);
-    finalConfig = self.__resolveConfigTemplates(event, config);
-    finalPayload = self.__resolveInput(event, config);
+    fullEvent = self.loadRemoteEvent(event)
```

Comment:
Since I'm including `loadRemoteEvent` in `loadNestedEvent`, perhaps we should make `loadRemoteEvent` a private function.